.github/workflows/qiita-plugin-ci.yml

```diff
@@ -101,9 +101,10 @@ jobs:
 
           export QIITA_SERVER_CERT=`pwd`/qiita-dev/qiita_core/support_files/server.crt
           export QIITA_CONFIG_FP=`pwd`/qiita-dev/qiita_core/support_files/config_test_local.cfg
+          export ENVIRONMENT="source /home/runner/.profile; conda activate qp-spades"
           pip install -U pip
           pip install .
-          configure_qp_spades --env-script "source /home/runner/.profile; conda activate qp-spades" --server-cert $QIITA_SERVER_CERT
+          configure_qp_spades --env-script 'source /home/runner/.profile; conda activate qp-spades; export ENVIRONMENT="source /home/runner/.profile; conda activate qp-spades"' --server-cert $QIITA_SERVER_CERT
```

Comment:
Not necessary here, but I wonder about doing something like `--env-script foo.sh`?

Reply:
Well, in theory (not sure about practice because I have never tried) we could do that as long as foo.sh exists as a script in the environment.

Reply:
ohhhh sorry, i didn't see this was in the GH actions. Ya it could work if defined upfront, but I think tweaking GH actions is an out of scope issue here